ESLint/lombiq-base.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
             { 'allowForLoopAfterthoughts': true }
         ],
 
-        'linebreak-style': ['warn', 'windows'],
+        'linebreak-style': 'off',
```

Comment:
No, unfortunately. It can only be "windows", "unix" or "off". [ESLint suggests forcing LF in gitattributes](https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems) for git users, but we are past that option.